packages/vat-data/test/test-scalar-only-keys.js

```diff
@@ -1,9 +1,9 @@
 // From https://github.com/Agoric/agoric-sdk/pull/6903#discussion_r1098067133
 
-import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import '@agoric/swingset-liveslots/tools/prepare-test-env.js';
 
+import test from 'ava';
```

Comment:
I'm not sure I like breaking out the Ava import separately.  Normally it doesn't matter much what order you put import statements in, but in this case I think it does.  Requiring the explicit Ava import opens up an error opportunity.

Reply:
Only prepare-test-env has side-effects. `import test from 'ava'` has none, so it can go anywhere.

Our pattern of getting `test` from a module that has side effects is one I assert we should deprecate: https://github.com/endojs/endo/issues/1467